modules/ding_dashboard/ding_dashboard.admin.inc

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * @file
+ * Editorial dashboard administrative interface.
+ */
+
+/**
+ * Editorial dashboard administrative block.
+ *
+ * Creates an administrative block on the configuration page for grouping
+ * various settings.
+ *
+ * @return string
+ *   HTML for the block.
+ */
+function ding_dashboard_admin_menu_block_page() {
+  $item = menu_get_item();
+  $content = system_admin_menu_block($item);
+
+  if ($content) {
+    $output = theme('admin_block_content', array('content' => $content));
+  }
+  else {
+    $output = t('You do not have any administrative items.');
+  }
+
+  return $output;
+}
+
+/**
+ * Content type selection form.
+ *
+ * Interface for selecting node types which should be a part
+ * of teh editorial system.
+ */
+function ding_dashboard_admin_content_types($form, $form_state) {
+  $node_types = node_type_get_types();
+  $node_bundles = array();
+
+  foreach ($node_types as $type) {
+    $node_bundles[$type->type] = $type->name;
+  }
+
+  $form['editorial_content_types'] = array(
```

Comment:
Please prefix both the form indexes and the variables store in "variables" table with module name (ding_).